### DISCUSSION TITLE
We are almost reaching the limit of Terraform Cloud free plan

### DISCUSSION THREAD

**Discussion by smorimoto** [asdf-community/.github#59] [General]
We now manage this hyper-scale plugin registry on top of HashiCorp's very generous Terraform Cloud free plan. We've been operating on a free plan, but the free plan has a limit of 500 resources and we're already using about 450. With two more plugins, we won't be able to stay there anymore. I would consider moving to some sort of object storage, but Terraform Cloud is still the best option anyway. I'd like to explore options to continue using it, but it doesn't seem as cheap as I thought. Does anyone have any good ideas?

**Comment by smorimoto** [comment]
@radditude I'm just wondering if maybe you can help us out. I tried to negotiate that through a support ticket, but they only responded like robots, and I'm now starting to suspect they're LLM.

**Reply by radditude** [reply]
I'll see what options we have!

**Reply by smorimoto** [reply]
That’s super helpful! Thank you so much!

**Reply by radditude** [reply]
HashiCorp doesn't offer any programs that would help here, unfortunately! However, that 500 resource limit is not a hard limit - once an organization is over the limit, they can keep using TFC, they're just capped at a certain number of applies per month and lose access to "extra" features like run tasks and Sentinel. So depending on our usage, we may be fine to keep using TFC even beyond the 500 resource limit.

**Reply by smorimoto** [reply]
That's a bit of a bummer, but I didn't know I could use TFC beyond 500 resources. I'll stick with it for a now.
